lib/utils.pm

```diff
@@ -2391,10 +2391,10 @@ sub ensure_ca_certificates_suse_installed {
     if (script_run('rpm -qi ca-certificates-suse') == 1) {
```

Comment:
let's just use https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/repo_tools.pm#L487 and we can throw all this away:
```
 if (script_run('rpm -qi ca-certificates-suse') == 1) {
my $distversion = generate_version();
 diag "CA folder: $distversion";
  zypper_call("ar --refresh https://download.opensuse.org/repositories/SUSE:/CA/$distversion/SUSE:CA.repo");
............
```